src/views/ConfigurationVideoView.vue

```diff
@@ -30,6 +36,21 @@
           hint="IP Addresses of the Vehicle allowed to be used for the WebRTC ICE Routing. Usually, the IP of the tether/cabled interface. Blank means any route. E.g: 192.168.2.2"
         />
       </div>
+
+      <div class="flex w-[30rem] flex-wrap">
+        <v-combobox
+          v-model="allowedIceProtocols"
+          multiple
+          :items="[ 'udp', 'tcp', 'tcp/udp' ]"
```

Comment:
why 'tcp/udp' if both 'udp' and 'tcp' can be selected?

Reply:
It was just testing, already removed in the new push